frontend/public/components/modals/_modals.scss

```diff
@@ -60,11 +60,19 @@
   display: flex;
   flex-direction: column;
   height: 100%;
+  position: relative;
   @media(min-width: $screen-sm-min) {
     // Desktop only responsive max-height allows modal to adjust to content height and enable scroll shadows if needed.
     max-height: calc(100vh - 60px);
+    // Dropdown workaround:  use when modal content can expand (taints, tolerations)
+    &--accommodate-dropdown .modal-body-inner-shadow-covers {
+      padding-bottom: 100px;
+    }
+    // Dropdown workaround: use when modal content is short and cannot expand
+    &--no-inner-scroll .modal-body {
+      overflow-y: visible ;
```

Comment:
This can cause an overlay issue 
<img width="780" alt="Screen Shot 2019-04-11 at 11 43 16 AM" src="https://user-images.githubusercontent.com/1874151/55972077-c6905e80-5c50-11e9-815e-e2461bbb2386.png">

Reply:
Yeah, but only if the viewport has a height less than 500px.  I opted for both fixes because, although not perfect by any means, they are less bad than the original bug.  I think we need to revisit this entirely in 4.2.